projects/plugins/protect/src/js/components/firewall-page/index.jsx

```diff
@@ -436,12 +494,54 @@ const FirewallPage = () => {
 						) }
 					</Text>
 					{ jetpackWafIpList && (
-						<div className={ styles[ 'edit-manual-rules-section' ] }>
-							<Text variant={ 'body-small' } mt={ 2 }>
-								{ '' === jetpackWafIpAllowList &&
-									'' === jetpackWafIpBlockList &&
-									__( 'No manual rules are being applied.', 'jetpack-protect' ) }
-							</Text>
+						<div className={ styles[ 'toggle-section__details' ] }>
+							<div className={ styles[ 'manual-rules-stats' ] }>
+								{ ipAllowListCount === 0 && ipBlockListCount === 0 && (
+									<Text
+										className={ styles[ 'manual-rules-stats__no-rules' ] }
+										variant={ 'body-small' }
+										mt={ 2 }
+									>
+										{ __( 'No manual rules are being applied.', 'jetpack-protect' ) }
+									</Text>
+								) }
+								{ ipBlockListCount > 0 && (
+									<Text
+										className={ styles[ 'manual-rules-stats__block-list-count' ] }
+										variant={ 'body-small' }
+										mt={ 2 }
+									>
+										{ sprintf(
+											// translators: placeholder is a number of blocked IP addresses i.e. "5 IPs are being blocked".
+											_n(
+												'%s IP is being blocked. ',
+												'%s IPs are being blocked. ',
+												ipBlockListCount,
+												'jetpack-protect'
+											),
+											ipBlockListCount === 1 ? 'One' : ipBlockListCount
+										) }
+									</Text>
+								) }
+								{ ipAllowListCount > 0 && (
```

Comment:
```suggestion
												'%s IP is being blocked.',
												'%s IPs are being blocked.',
												ipBlockListCount,
												'jetpack-protect'
											),
											ipBlockListCount === 1 ? 'One' : ipBlockListCount
										) }
									</Text>
								) }
								{ ipBlockListCount > 0 && ipAllowListCount > 0 && ' ' }
								{ ipAllowListCount > 0 && (
```
Instead of putting the empty space in the translated string, I would recommend pulling it out and manually adding the space between the two sentences when applicable.

I know it looks a little more messy, but then we are only rendering the space when necessary, and the presence of it isn't dependent on the translated string.